presentation-core/src/main/java/tachiyomi/presentation/core/components/SettingsItems.kt

```diff
@@ -115,6 +116,30 @@ fun SortItem(label: String, sortDescending: Boolean?, onClick: () -> Unit) {
     )
 }
 
+@Composable
+fun NondirectionalSortItem(label: String, enabled: Boolean, enabledIcon: ImageVector, onClick: () -> Unit) {
```

Comment:
This should be made into `BaseSortItem` that takes in `label: String`, `icon: ImageVector?` and `onClick: () -> Unit`. SortItem should use `BaseSortItem`. Then in `LibrarySettingsDialog` create the random sort item using `BaseSortItem`